### PR TITLE
add Chinese (Simplified) localization

### PR DESCRIPTION
**Pull request by speadup**
add Chinese (Simplified) localization
build test on msys2
test on Windows 10 x64

### CODE REVIEW

**Comment by selvanair** [comment]
@speadup This is a nice addition and we want to get this merged. Know anyone who could give a review? @TSLCS ?

File: res/openvpn-gui-res-zh-hans.rc [hunk 1]
```diff
@@ -0,0 +1,480 @@
...
+
+/* Challenge Response Dialog */
+ID_DLG_CHALLENGE_RESPONSE DIALOG 6, 18, 212, 72
+STYLE WS_SIZEBOX | WS_POPUP | WS_VISIBLE | WS_CAPTION | DS_CENTER | DS_SETFOREGROUND
+CAPTION "OpenVPN - 质询回应"
+FONT 8, "Microsoft Sans Serif"
+LANGUAGE LANG_CHINESE, SUBLANG_CHINESE_SIMPLIFIED
+BEGIN
+    LTEXT "", ID_TXT_DESCRIPTION, 6, 9, 208, 10
+    LTEXT "回应:", 0, 6, 30, 50, 10
```
Comment (huiwenliu95) [inline res/openvpn-gui-res-zh-hans.rc]: 答复 maybe better?

**Comment by speadup** [comment]
@huiwenliu95 thank you! updated

**Comment by speadup** [comment]
@selvanair squashed，thank you！

**Comment by selvanair** [comment]
Looks good. Merging based on @huiwenliu95 ' s review. Thanks.

**Pull request merged**